src/Thoth.Json/Decode.fs

```diff
@@ -265,22 +241,17 @@ module Decode =
             if Helpers.isNumber value then
                 Helpers.asFloat value |> decimal |> Ok
             elif Helpers.isString value then
-                try
-                    // System.Decimal.Parse (Helpers.asString value, CultureInfo.InvariantCulture)
-                    // Remove CultureInfo.InvariantCulture for now, otherwise Fable output a warning
-                    System.Decimal.Parse (Helpers.asString value)
-                    |> Ok
-                with
-                    | ex ->
-                        (path, BadPrimitiveExtra("a decimal", value, ex.Message)) |> Error
+                match System.Decimal.TryParse (Helpers.asString value) with
+                | true, x -> Ok x
+                | _ -> (path, BadPrimitive("a decimal", value)) |> Error
             else
                 (path, BadPrimitive("a decimal", value)) |> Error
 
     let datetime : Decoder<System.DateTime> =
         fun path value ->
             if Helpers.isString value then
                 match System.DateTime.TryParse (Helpers.asString value) with
-                | true, x -> Ok x
+                | true, x -> x.ToUniversalTime() |> Ok
```

Comment:
This is probably a breaking change

Reply:
Yes, maybe. IMHO it's easier for users to expect dates to be deserialized as universal time and then they can change it to the local time of the machine if necessary. I always found the behavior in .NET and Newtonsoft.Json confusing. What do you think?

Reply:
See #76

Reply:
To be honest I don't use much .NET so I don't know their behavior. 

I am just not sure about this change from a user point of view because it can lead to subtile bugs in their code.
We will make sure to have it mentioned in the documentation